doc_classes/BTCallMethod.xml

```diff
@@ -10,6 +10,9 @@
 	<tutorials>
 	</tutorials>
 	<members>
+		<member name="include_delta" type="bool" setter="set_include_delta" getter="is_delta_included" default="false">
```

Comment:
A small tip: 
If you compiled the editor, you can run `bin/godot.* --doctool .` from a terminal to generate the class documentation structure, and then you only need to add your descriptions. It also does formatting and sorting for the members.